tests/unit/__snapshots__/documentation.spec.ts.snap

```diff
@@ -3683,6 +3683,80 @@ servers:
 "
 `;
 
+exports[`Documentation > Feature #1869: Top level transformations > should handle in request and response 1`] = `
+"openapi: 3.1.0
+info:
+  title: Testing top level transformations
+  version: 3.4.5
+paths:
+  /v1/test:
+    get:
+      operationId: GetV1Test
+      parameters:
+        - name: user_id
+          in: query
+          required: true
+          description: GET /v1/test Parameter
+          schema:
+            type: string
+      responses:
+        "200":
+          description: GET /v1/test Positive response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    const: success
+                  data:
+                    format: any
```

Comment:
this happens because the transformation result is not a primitive type, and that's not handled

Reply:
fixed